docs/src/man/getting_started.md

```diff
@@ -573,6 +594,40 @@ a function object that tests whether each value belongs to the subset
     More details on copies, views, and references can be found
     [here.](https://juliadata.github.io/DataFrames.jl/stable/lib/indexing/#getindex-and-view-1)
 
+#### Selecting Rows with `filter`
+
+We have seen above how to subset a `DataFrame` to several criteria, involving multiple columns, by supplying a logical vector to the first dimension. For instance, in the following we want to subset to all rows where `x > 2` and where `a == 'c'`:
```

Comment:
breaking long line
```suggestion
We have seen above how to subset a `DataFrame` to several criteria, involving multiple columns,
by supplying a logical vector to the first dimension. For instance, in the following we want to subset
to all rows where  `x > 2` and where `a == 'c'`:,
```

Reply:
Since we have provided examples above, it redundant to introduce a new example here using the other syntax: better show how to reproduce the example from above using `filter`.

Reply:
I think it's quite instructive to have this example here instead of the previous one (which only has integer cols). looks more like a real world case. you have a strong opinion on that one?

Reply:
I think that in the manual it is not a problem to have redundancy.

Reply:
actually this exact pattern is one of the most common questions I get.